src/app/course/components/page-list/page-list.component.ts

```diff
@@ -30,8 +25,9 @@ export class PageCoursesComponent implements OnInit {
 		this._coursesService.getList({search}).then(this._mapCourses.bind(this));
 	}
 
+
 	private _mapCourses(items: ICourse[]) {
 		this.listCourses = [];
-		map(items, (item: CourseProps) => this.listCourses.push(new Course(item)))
+		map(items, (item: TCourse) => this.listCourses.push(new Course(item)))
```

Comment:
This case should be better for performance:
this.listCourses = map(items, (item: CourseProps) => new Course(item));